src/server/pachyderm_test.go

```diff
@@ -689,7 +689,7 @@ func TestPipelineThatOverwritesFile(t *testing.T) {
 	require.Equal(t, 1, len(outCommits))
 	var buffer2 bytes.Buffer
 	require.NoError(t, c.GetFile(outRepo.Name, outCommits[0].Commit.ID, "file", 0, 0, "", nil, &buffer2))
-	require.Equal(t, "foo\nfoo\nfoo\nfoo\nfoo\nfoo\n", buffer2.String())
+	require.Equal(t, "foo\nfoo\nfoo\n", buffer2.String())
```

Comment:
The behavior of `PutFile` remains the same. What changes is the behavior of `>` because `>` starts by sending a truncate request to the file and we now correctly implementing that. This is the semantics we'd talked about previously where to overwrite a file you do `>` and to append to it you do `>>` just like shell.
I tried to go through the tests and fix them so they kept with the original intent, since this test is for overwriting I figured that it was more reasonable to use `>` instead of `>>` and just change the output we expect.